src/AddIns/Uno.UI.GooglePlay/Uno.UI.GooglePlay.netcoremobile.csproj

```diff
@@ -0,0 +1,67 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFrameworks>$(NetAndroidPreviousAndCurrent)</TargetFrameworks>
+	</PropertyGroup>
+
+	<Import Project="../../targetframework-override.props" />
+
+	<PropertyGroup>
+		<NoWarn>$(NoWarn);Uno0001</NoWarn>
+		<Deterministic>true</Deterministic>
+		<Nullable>enable</Nullable>
+		<AssemblyName>Uno.UI.GooglePlay</AssemblyName>
+		<RootNamespace>Uno.UI.GooglePlay</RootNamespace>
+
+		<DefineConstants Condition="'$(UNO_UWP_BUILD)'!='true'">$(DefineConstants);HAS_SKOTTIE</DefineConstants>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<None Remove="buildTransitive\*.GooglePlay.targets" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<Content Include="buildTransitive\*.GooglePlay.targets">
+			<PackagePath>build</PackagePath>
+			<Pack>true</Pack>
+		</Content>
+	</ItemGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Xamarin.Google.Android.Play.Core" Version="1.10.3.14" />
+
+		<ProjectReference Include="..\..\Uno.Foundation.Logging\Uno.Foundation.Logging.csproj" />
+		<ProjectReference Include="..\..\Uno.UI\Uno.UI.netcoremobile.csproj" />
+	</ItemGroup>
+
+	<Import Project="..\..\SourceGenerators\Uno.UI.SourceGenerators\Content\Uno.UI.SourceGenerators.props" />
+
+	<Target Name="_CleanupReferences" BeforeTargets="CoreCompile">
+		<!-- Required when building under VS to remove analyzer references coming from the Uno.UI/Uno.WinUI transitive references -->
+		<ItemGroup>
+			<Analyzer Remove="@(Analyzer)" Condition="'%(Analyzer.NugetPackageId)'=='Uno.UI'" />
+			<Analyzer Remove="@(Analyzer)" Condition="'%(Analyzer.NugetPackageId)'=='Uno.WinUI'" />
+		</ItemGroup>
+	</Target>
+
+	<Target Name="_UnoToolkitOverrideNuget" AfterTargets="AfterBuild" DependsOnTargets="BuiltProjectOutputGroup" Condition="'$(UnoNugetOverrideVersion)'!=''">
+
+		<PropertyGroup>
+			<_OverrideTargetFramework>$(TargetFramework)</_OverrideTargetFramework>
+			
+			<!-- Current nuget publishing uses netX.0-android30.0 -->
```

Comment:
This package is likely going to be without platform version using MSBuild packaging